c/include/libsbp/sbas.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
```

Comment:
change year to 2018

Reply:
These come from the template. I think it's better to update these in separate PR and preferably by the repository POC.

Reply:
In other words I'd ping @denniszollo about this.

Reply:
fair enough then - they were only nits - looks good